code/api/src/modules/user/types.js

```diff
@@ -12,6 +12,7 @@ const UserType = new GraphQLObjectType({
     email: { type: GraphQLString },
     password: { type: GraphQLString },
     role: { type: GraphQLString },
+    stylePreference: { type: GraphQLString },
```

Comment:
Im curious if we will eventually have to turn this to an Array

Reply:
Excellent job adjusting our user Type – this new property is exactly what we need to access the user's stylePreference on login.